engine/apps/public_api/serializers/incidents.py

```diff
@@ -29,6 +29,7 @@ class Meta:
             "resolved_at",
             "acknowledged_at",
             "title",
+            "permalink",
```

Comment:
I sense a n+1 issue here since permalink depends on `SlackMessage` model, please check if select_related / prefetch_related is helpful here.

Reply:
Good point! I noticed in the "internal" api serializer we don't add `permalink` to either `SELECT_RELATED` or `PREFETCH_RELATED`. Should the public api serializer deviate from this?

Reply:
Yes, the internal api is a little bit different and only returns permalinks for requests to particular alert group. The `AlertGroupListSerializer` doesn't have a `permalink` field so it's not in `SELECT_RELATED` or `PREFETCH_RELATED`. For public api we only have one serialiser so we probably should add it here.